QUEST.tsv

```diff
@@ -56,14 +56,14 @@ QUEST_20150317_000055	I don't know where to start. {nl}Have you heard anything?
 QUEST_20150317_000056	By royal command, we've come to this forest to investigate some strange events. {nl}Witnesses have reported something with a big face and bony fingers. {nl}What in the world could that be?
 QUEST_20150317_000057	In the past, pilgrims travelled through this forest. {nl}But now, only soldiers gather here due to the monster presence.
 QUEST_20150317_000058	Up until four years ago, the Goddesses often blessed us with signs of their presence. {nl}But now, even our combined prayers don't bring so much as a tremor. {nl}Have they truly abandoned us?
-QUEST_20150317_000059	Vacenin personally goes to each region to deal with things. {nl}He gets uncomfortable when he stays in one place for too long.
-QUEST_20150317_000060	Do you plan on staying or enlisting? {nl}If you're interested, I'll gladly recommend you to the head commanders.
+QUEST_20150317_000059	Vacenin personally goes to each region to deal with things. {nl}He gets uncomfortable when he stays in the city.
+QUEST_20150317_000060	Do you plan on staying or enlisting? {nl}If you're interested, I can see about moving you up.
```

Comment:
The original translation is correct, the new one seems like a mistranslation, confirmed by SHUcream on discord. Please revert.